File: Stack/python/stack_operations.py
from stack_class import Stack

stack = Stack() # iniilize the stack object
 
stack.push(10)  # push values to the stack
stack.push(20) 
stack.push(22) 

stack.printStack() # print the stack

stack.pop()   # remove the last element from the list

stack.printStack() # print the stack

stack.isEmpty()# check is Empty on the stack

stack.peek() # print the top most element in the stack

File: Stack/nodejs/stack_operations.js
const Stack = require('./stack_class.js')

var stack = new Stack();  // iniilize the stack object
stack.push("value 1");  // push values to the stack
stack.push("value 2");  
stack.push("value 3");  

console.log(stack.printStackItems())  // print all items in the stack 

console.log(stack.isEmpty()) // check the stack is empty

console.log(stack.pop()) // set, and remove the top most element from the stack

console.log(stack.peek()) // get the top most value fom the stack (do not delete the value)
